code/src/java/pcgen/core/Equipment.java

```diff
@@ -2349,19 +2349,6 @@ public String getWieldName()
 	}
 
 	/**
-	 * Description of the Method
-	 * 
-	 * @param aPC The PC that has this Equipment
-	 * 
-	 * @return Description of the Return Value
-	 */
-	public Integer acCheck(final PlayerCharacter aPC)
-	{
-		return Math.min(getSafe(IntegerKey.AC_CHECK)
-			+ (int) bonusTo(aPC, "EQMARMOR", "ACCHECK", true), 0);
```

Comment:
IMO this method is sophisticated enough to warrant being refactored the way it was. So having it "unrefactored" seems kind of backwards to me from a code management standpoint. I don't mind it being removed from Equipment but at the very least I think it should be moved to a utility class.